Makefile

```diff
@@ -1,5 +1,6 @@
 KUBERNETES_VERSION=1.9
 DIND_CLUSTER_SCRIPT=dind-cluster-v$(KUBERNETES_VERSION).sh
+DIND_CLUSTER_RELEASE_VERSION=$$(if [ "$(KUBERNETES_VERSION)" = "1.14" ]; then echo v0.2.0; else echo v0.1.0; fi)
```

Comment:
Sucks that we have to do this :( Hopefully I'll get around to simplifying the test setup sometime soon ;)